src/cr/cube/measures/new_pairwise_significance.py

```diff
@@ -83,6 +123,17 @@ def pairwise_indices(self):
             significance = np.logical_and(self.t_stats < 0, significance)
         return [tuple(np.where(sig_row)[0]) for sig_row in significance]
 
+    @lazyproperty
+    def pairwise_indices_scale_mean(self):
+        # Returns a list of n tuples where n is the number of the columns
+        # of the slice. Each tuple contains the indices of the significant
+        # column compared with the col ix where ix is the index of the tuple
+        # in the list. e.g [(), (), (1,), (0, 1, 2)]
+        significance = self.p_vals_scale_means < self._alpha
+        if self._only_larger:
+            significance = np.logical_and(self.t_stats_scale_means < 0, significance)
```

Comment:
```suggestion
            significance = np.logical_and(self.t_stats_scale_means > 0, significance)
```
I would have thought it was `t = observed - expected / se` in both cases, but this flip should do the trick I was trying to communicate.